src/test/resources/application-local.yml

```diff
@@ -42,7 +41,7 @@ no.nav.security.jwt:
     saf:
       token-endpoint-url: http://metadata
 
-TOKEN_X_WELL_KNOWN_URL: http://localhost
+TOKEN_X_WELL_KNOWN_URL: https://fakedings.intern.dev.nav.no/fake/.well-known/openid-configuration
 TOKEN_X_PRIVATE_JWK: '{
```

Comment:
Trengs private-jwk nå? 🤔

Reply:
Den prøver å parse JWK-key, med mindre jeg overstyrer alt av client registration som står i `application.yml`, så tenker det er like greit med en dummy jwk for å ha mindre config i `application-local` slik at det forhåpentligvis blir litt lettere å forstå.